package.json

```diff
@@ -32,14 +32,14 @@
     "babel-loader": "6.2.4",
     "babel-preset-es2015-native-modules": "6.9.0",
     "babel-preset-react": "6.11.1",
-    "electron-builder": "5.14.2",
-    "electron-prebuilt": "1.2.8",
+    "electron-builder": "5.16.0",
+    "electron-prebuilt": "1.3.0",
```

Comment:
Hmm, I'll have to take a look. Can the one in `app/plugins` be computed from the `package.json` somehow instead?